manifests/s/schild-report/schild-report/16.16.3/schild-report.schild-report.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: schild-report.schild-report
+PackageVersion: 16.16.3
+InstallerLocale: de-DE
+MinimumOSVersion: 10.0.0.0
+InstallerType: msi
+Scope: user
+InstallModes:
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v16.16.3/SchemaCrawler-16.16.3.msi
```

Comment:
Uhhhm, this is the wrong url

Reply:
https://github.com/microsoft/winget-pkgs/pull/36264#issuecomment-980830866

It got Azure Pipeline Passed and Validation Completed labels and got merged.